docs/demo/gaiden-css/generic/headings/demo.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="maximum-scale=1,width=device-width,initial-scale=1,user-scalable=0">
+ <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+  <style>
+    nav {
+      padding: 20px;
+    }
+
+  </style>
+  <title>Headings</title>
+</head>
+
+<body>
+    <h1 class="" arial-level="1" role="heading">
+      H1 Title Gaiden
+      <small>Sub Title Gaiden</small>
```

Comment:
If we have:
<Section>
<H1> Tittle </ h1>
<H2> Subtitle </ h2>
</ Section>

For h2 without less relevant than h1 this no longer defines as subtitle?
I think it's worth seeing this with the SEO guys.